packages/@okta/vuepress-site/docs/guides/oie-embedded-common-refresh-tokens/main/ios/refreshusingthesdk.md

```diff
@@ -0,0 +1,26 @@
+## Option 1: Refresh using the Swift SDK
+
+The Swift SDK has built-in support for refreshing tokens through
+the use of the `Token` object's `refresh` method.
```

Comment:
```suggestion
the use of the `token` object's `refresh` method.
```

Reply:
rewording to say: "the use of the `Token` class's `refresh` method." 

The class is defined with a capital T.